packages/app/cypress/e2e/integration/runs.spec.ts

```diff
@@ -1,80 +1,249 @@
-describe('App: Runs Page', () => {
+import type { Interception } from '@packages/net-stubbing/lib/external-types'
+
+describe('App: Runs', () => {
   beforeEach(() => {
     cy.scaffoldProject('component-tests')
     cy.openProject('component-tests')
     cy.startAppServer()
   })
 
-  it('resolves the runs page', () => {
-    cy.loginUser()
-    cy.visitApp()
-    cy.wait(1000)
-    cy.get('[href="#/runs"]').click()
-    cy.get('[data-cy="runs"]')
+  context('Runs Page', () => {
+    it('resolves the runs page', () => {
+      cy.loginUser()
+      cy.visitApp()
+      cy.wait(1000)
```

Comment:
Is the wait still required here? Could we increase the timeout for the following `get` instead?